tests/unit/OptionsListUtilsTest.js

```diff
@@ -1516,35 +1518,405 @@ describe('OptionsListUtils', () => {
+                name: 'Travel: Meals: Lunch',
+            },
+            'Cars: Mercedes-Benz': {
+                enabled: true,
+                name: 'Cars: Mercedes-Benz',
+            },
+            Food: {
+                enabled: true,
+                name: 'Food',
+            },
+            'Food: Meat': {
+                enabled: true,
+                name: 'Food: Meat',
+            },
+            'Travel: Meals: Dinner': {
+                enabled: false,
+                name: 'Travel: Meals: Dinner',
+            },
+            'Food: Vegetables': {
+                enabled: false,
+                name: 'Food: Vegetables',
+            },
+            Restaurant: {
+                enabled: true,
+                name: 'Restaurant',
+            },
+            Taxi: {
+                enabled: false,
+                name: 'Taxi',
+            },
+            'Food: Milk': {
+                enabled: true,
+                name: 'Food: Milk',
+            },
+            'Travel: Meals': {
+                enabled: true,
+                name: 'Travel: Meals',
+            },
+            'Travel: Meals: Breakfast': {
+                enabled: true,
+                name: 'Travel: Meals: Breakfast',
+            },
+            'Cars: Audi': {
+                enabled: true,
+                name: 'Cars: Audi',
+            },
+        };
+        const result2 = [
+            {
+                enabled: true,
+                name: 'Cars: Audi',
+            },
+            {
+                enabled: false,
+                name: 'Cars: BMW',
+            },
+            {
+                enabled: true,
+                name: 'Cars: Mercedes-Benz',
+            },
+            {
+                enabled: false,
+                name: 'Medical',
+            },
+            {
+                enabled: true,
+                name: 'Travel: Meals',
+            },
+            {
+                enabled: true,
+                name: 'Travel: Meals: Breakfast',
+            },
+            {
+                enabled: false,
+                name: 'Travel: Meals: Dinner',
+            },
+            {
+                enabled: true,
+                name: 'Travel: Meals: Lunch',
+            },
+            {
+                enabled: true,
+                name: 'Food',
+            },
+            {
+                enabled: true,
+                name: 'Food: Meat',
+            },
+            {
+                enabled: true,
+                name: 'Food: Milk',
+            },
+            {
+                enabled: false,
+                name: 'Food: Vegetables',
+            },
+            {
+                enabled: true,
+                name: 'Restaurant',
+            },
+            {
+                enabled: false,
+                name: 'Taxi',
+            },
+        ];
+
+        expect(OptionsListUtils.sortCategories(categoriesIncorrectOrdering)).toStrictEqual(result);
```

Comment:
This is a nit, but maybe it would be more readable if moved before the `result2` definition?

Reply:
I am used to define variables, then run tests with them. I already used this strategy before at Expensify e.g. https://github.com/Expensify/App/blob/main/tests/unit/OptionsListUtilsTest.js#L670. 

<img width="771" alt="Screenshot 2023-10-18 at 12 47 58" src="https://github.com/Expensify/App/assets/57314631/82a4e7de-3314-4c76-a842-a357a70f2c3c">

Reply:
Sure